docs/community/intro.md

```diff
@@ -4,12 +4,28 @@ sidebar_label: 🧐 How It Works
 description: Overview of what SourceCred is and how it works
 ---
 
-Everything people do to support a project—like writing code, filing bug reports, organizing meetups, or even mediating hard discussions—is considered a contribution.
+SourceCred is an algorithm that evaluates the value of, and shows the relationships between, the contributions that collaborators make to a project. 
 
-These contributions are connected to each other based on their relationships. For example, a pull request may fix a particular bug, and a code review may review that pull request.
+Anything a contributor does to support a project they're working on - from the technical (like writing code or filing bug reports), to the logistical (like organizing events or work sessions), to the emotional (like mediating difficult discussions or helping someone feel welcome) is considered a contribution that has some amount of value. 
 
-This network of contributions and relationships form a contribution graph. SourceCred applies the [PageRank algorithm](https://en.wikipedia.org/wiki/PageRank)  to this graph, generating a score, called cred for every contribution.
+Each of these contributions made to the project are connected to each other in a large graph of "nodes" (the contributions and contributors) and "edges" (the lines that connect nodes together). This graph shows the way each contribution and contributor in the project relates to the other nodes it affects or supports. 
 
-The basic idea is: a contribution earns cred if it is connected to other contributions that earn lots of cred. Contributors are connected to the contributions they've helped with, which means that they also earn cred.
+This large graph of every contribution made to a project and the ways those contributions are connected is then analized by a [PageRank algorithm](https://en.wikipedia.org/wiki/PageRank) which generates a score for each contribution and contributor based on how connected they are to the rest of the graph. The more valuable the contribution is, the higher it's score. 
```

Comment:
suggest changing `This large graph of..` to `This "contribution graph"`.

We'll likely want a glossary page for "contribution graph".

Reply:
Since this is an introduction that assumes people are not familiar with SC or fluent in math/tech topics, my intention was to take phrases like "Contribution Graph" and give a longer form explanation that builds on the concepts I'd introduced just before this paragraph. I didn't want to assume the audience knows our terminology in the first document they put their eyes on. 

However, I could maybe take a moment in the paragraph before the one you're commenting about to establish the phrase "contribution graph" so that people can start getting used to our terminology.  Or even just add something like "..ways those contributions are connected (aka a "contribution graph") is then..."

Do you agree that it's important to explain our concepts in full in this document, or would you prefer to be more brief and assume the newcomers reading this document already have enough understanding to make those intuitive/terminology leaps?

Reply:
The contribution graph is a super essential topic, so I'd be happy to have an overview of what it is in an intro to SC document. E.g. having a paragraph introducing the concept. I like the idea of using one of the drawings that you made to illustrate the concept.